src/components/Checkbox/index.tsx

```diff
@@ -21,9 +21,15 @@ export const getHoverStyle = (props: any) => `
 	& label:hover input:not([disabled]) + div {
 		border-color: ${props.theme.colors.tertiary.main}!important;
 	}
-	& label:hover input[checked]:not([disabled]) + div {
-		border-color: ${props.theme.colors.primary.dark}!important;
-		background: ${props.theme.colors.primary.dark};
+	${
```

Comment:
Grommet stopped applying the `checked` attribute on both radio buttons and checkboxes, so we pass the checked prop instead. This wouldn't work for uncontrolled components, but we always have the components as controlled so it doesn't matter.